Makefile.am

```diff
@@ -14,3 +15,21 @@ coverage: check
 	@echo -e "libica coverage";
 	@echo -e "---------------\n";
 	cd ${top_builddir}/src && gcov .libs/*.gcda
+
+if ICA_FIPS
+check: all
+	@echo make all and all-local before make check because of .hmac file
+
+all-local:
```

Comment:
Shouldn't this be `all-hook`?  From what I know, -local targets are executed **before** the corresponding target and -hook targets are executed **after** the target.  In our case, we want the all target to build the library and then create the .hmac file.

Reply:
Also this rule should not be in this Makefile.am, but in src/Makefile.am as an all-hook.  Then I think it is executed at the correct place and you do not need the ``check: all`` artificial dependency.

Reply:
ok, I moved this target to src/Makefile.am:

- The `check: all` dependency is now in fact superfluous
- But the all-local cannot be replaced by all-hook

I realized that VERSION is defined in src/makefile.am as `VERSION = 3:7:0` and I wonder why colons are used here?
So I introduced a VERSION1 temporarily as `VERSION1=3.7.0`, but this is not what we want at the end.